lib/src/network_service.dart

```diff
@@ -40,7 +41,8 @@ class NetworkService {
     return await _post(
             url,
             {
-              "user": user.toPrivacySensitiveJson(),
+              "user": user.toPrivacySensitiveJson()
+                ..addAll(_options.environment?.toJson() ?? {}),
```

Comment:
This is great for `initialize`, but now we arent setting environment on log events.  We need to make a copy of the user that comes in, add environment to it, so it gets sent with initialize and each event